deepchem/utils/save.py

```diff
@@ -116,6 +119,44 @@ def load_csv_files(filenames, shard_size=None, verbose=True):
         yield df
 
 
+def load_json_files(filenames: List[str],
+                    shard_size: Optional[int] = None):
```

Comment:
Is there a way to get the yield type into the type annotation? Sorry, still a mypy noob so not sure if there's a good way to do this :)

Reply:
Yes! I forgot to do this, but I believe the mypy preferred way to do this is to say `-> Iterator[type]` for whatever type is yielded. I'll add that in.